db/model/subject.js

```diff
@@ -720,6 +675,53 @@ module.exports = function subject(seq, dataTypes) {
             resolve(found.length === 1);
           }));
       }, // isWritableBy
+
+      setupParentFields() {
+        if (this.parentAbsolutePath || this.parentId) {
+          let key = null;
+          let value = null;
+          if (this.parentId) {
+            key = 'id';
+            value = this.getDataValue('parentId');
+          } else {
+            key = 'absolutePath';
+            value = this.getDataValue('parentAbsolutePath');
+          }
+
+          const whereObj = { where: {} };
+          if (key === 'absolutePath') {
+            whereObj.where[key] = { $iLike: value };
+          } else {
+            whereObj.where[key] = value;
+          }
```

Comment:
The scopes `id` and `absolutePath` were created to handle L691 to L697. Can you use them instead?

Reply:
It was using those originally - I changed it because there was an issue with case insensitivity. 
I changed it back and updated the scope to be case insensitive instead. Any objection to having the scope be case-insensitive for all queries?

Reply:
@shriramshankar @iamigo

Reply:
Makes sense using the case-insensitive comparison.